tensorboard/plugins/profile_redirect/tf_profile_redirect_dashboard/tf-profile-redirect-dashboard.ts

```diff
@@ -49,6 +49,15 @@ class TfProfileRedirectDashboard extends LegacyElementMixin(PolymerElement) {
     </div>
 
     <style>
+      :host {
+        display: block;
+        height: 100%;
+        left: 0;
+        position: absolute;
+        top: 0;
+        width: 100%;
```

Comment:
top/left can be dropped.

Optional: replacing this entire block with 
```
:host {
  display: flex;
}
```
seems like it would have the same effect, if you prefer avoiding pos: abs.

Reply:
FWIW, they really are not equivalent. 

![image](https://user-images.githubusercontent.com/2547313/120841759-c2a2b300-c520-11eb-9278-084a37707205.png)

Without global background color set, with flex, it would look incorrect (which was what I used to have). However, since I now set the global background, they appear the same thus incorporated your suggestion.